src/backlight/portfolio/strategy.py

```diff
@@ -1,11 +1,16 @@
 # Define how to create Portfolio
 from typing import List
-from backlight.portfolio.portfolio import Portfolio
+from backlight.portfolio.portfolio import (
+    Portfolio,
+    _get_forex_ratios,
```

Comment:
It should not be good to import private function. How about locating it in other place and make it public?

Reply:
Where do you suggest to add it? In datasource/utils.py?

Reply:
That is fine!

`datasource/utils.py`